p521/src/arithmetic/field.rs

```diff
@@ -101,14 +101,54 @@ impl FieldElement {
         Self::from_uint_unchecked(U576::from_u64(w))
     }
 
+    /// Converts the words into a byte array to be consumed by [`fiat_p521_from_bytes`]
+    const fn words_to_le_bytes_unchecked(words: [u64; 9]) -> [u8; 66] {
```

Comment:
This can go in the [`util` module](https://github.com/RustCrypto/elliptic-curves/blob/master/p521/src/arithmetic/util.rs) as a free/static function.

Also, since it's designed to operate on the saturated representation used by `crypto-bigint`, it would probably make sense for it to accept a `&U576` as an argument to avoid confusion with `fiat_p521_tight_field_element`.

Reply:
Done in https://github.com/RustCrypto/elliptic-curves/pull/945/commits/f0299fd0395c1997470d39b8ad1f2eb0c6fecf84